Jenkinsfile

```diff
@@ -52,12 +41,12 @@ def slackGithub(String message, String color = null) {
     slackSend channel: "#github", color: color, message: formattedMessage
 }
 
-def runIntegrationTest(String workspace, String moduleName) {
-    String modulePath = "${workspace}/grakn-test/${moduleName}"
+def runIntegrationTest(String moduleName) {
+    String modulePath = "${pwd()}/grakn-test/${moduleName}"
```

Comment:
Kind of find `workspace` easier to work with. I hate "embedded paths" but that's just me.

Reply:
Yeah, I'm not sure which is better. I'll try changing it back.